dynamic-scoring-framework/.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
```

Comment:
nit: these gha will not work here. You can utilize addon-contrib's gha to run test and e2e test.